swagger.yaml

```diff
@@ -155,5 +317,116 @@ components:
           format: date
         assignedPersons:
           type: array
-          items: 
-            type: integer
\ No newline at end of file
+          items:
+            type: integer
+
+    Leads:
+      type: object
+      description: Lead data from Pipedrive API
+      properties:
```

Comment:
All properties should have a description, also any where an id us being used if it is a uuid then you should set the format as uuid, this is also the case when defining the parameters schema for the endpoints that are using ids.